pkg/gits/bitbucket_server.go

```diff
@@ -658,7 +680,7 @@ func (b *BitbucketServerProvider) GetPullRequestCommits(owner string, repository
 	paginationOptions["start"] = 0
 	paginationOptions["limit"] = pageLimit
 	for {
-		apiResponse, err := b.Client.DefaultApi.GetPullRequestCommitsWithOptions(repository.Project, repository.Name, number, paginationOptions)
+		apiResponse, err := b.Client.DefaultApi.GetPullRequestCommitsWithOptions(strings.ToUpper(owner), repository.Name, number, paginationOptions)
```

Comment:
I am a bit confused by switching from 'Project' to 'owner'. Or in another case to 'organisation'. Has there been a change in the API or was the existing code never working?

Reply:
The original code didn’t work consistently, yeah.

Reply:
See my comment above for a bit more detail. Using `Project` in these API calls would only work consistently if the `repository` had been generated from the BBS provider directly, since otherwise `Project` isn’t set with the general purpose git url parsing stuff.

Reply:
> See my comment above for a bit more detail. Using Project in these API calls would only work consistently if the repository had been generated from the BBS provider directly, since otherwise Project isn’t set with the general purpose git url parsing stuff.

got you